### PR TITLE
[CI] Change double to single quotes and fix bugs

### PR DESCRIPTION
**Pull request by SaschaMann**
Thanks for nothing, workflow linter.

### CODE REVIEW

**Comment by SaschaMann** [comment]
The previous version had various other bugs that I didn't catch in #1582. I've tested this for pushes from the same repo, but not from a fork. However, I believe that should work because it falls back to an old version that worked (but I've been wrong about that so many times by now I don't even know)

File: .github/workflows/format-code.yml [hunk 1]
```diff
@@ -9,15 +9,10 @@ jobs:
...
-      - name: 'Check fork status'
-        id: fork_check
-        run: |
-          IS_FORK="$(jq '.pull_request.head.repo.fork' "$GITHUB_EVENT_PATH")"
-          echo "::set-output name=fork::$IS_FORK"
-          
+    steps:          
       - name: 'Setup SSH deploy key'
-        if: steps.fork_check.outputs.fork == "false"
+        # This will need to be adjusted when PRs are run against forks of the v3 repo
```
Comment (SaschaMann) [inline .github/workflows/format-code.yml:14]: In other words, if I want this to work on my own fork `SaschaMann/v3`, I'd have to change this to `SaschaMann`

**Comment by SaschaMann** [comment]
(not ready yet)

File: .github/workflows/format-code.yml [hunk 2]
```diff
@@ -9,15 +9,10 @@ jobs:
...
+        if: github.repository_owner == 'exercism'
```
Comment (ErikSchierboom) [inline .github/workflows/format-code.yml:15]: I like this new way of checking for forks! Much simpler.

Comment (SaschaMann) [reply .github/workflows/format-code.yml:15]: Unfortunately it doesn't work 😬
Since the action always runs in the v3 repo, it would always be `'exercism'`

Comment (ErikSchierboom) [reply .github/workflows/format-code.yml:15]: Ah, too bad! Would have been nice :)

**Pull request closed without merging**